llvm/test/Transforms/PGOProfile/memprof-undrift.test

```diff
@@ -0,0 +1,156 @@
+; REQUIRES: x86_64-linux
+
+; Make sure that we can undrift the MemProf profile and annotate the IR
+; accordingly.
+
```

Comment:
Maybe add the c++ source used in the comments?

Reply:
Fixed in the latest revision.